server/controllers/AdminController.js

```diff
@@ -0,0 +1,44 @@
+import bcrypt from 'bcryptjs';
+import models from '../database/models';
+import helpers from '../helpers';
+
+const {
+  responseMessage,
+} = helpers;
+const { User } = models;
+
+/**
+ * Admin controller class
+ * @class
+ */
+class AdminController {
+  /**
+   * Create a new user
+   * @name createUser
+   * @param {object} request
+   * @param {object} response
+   * @returns {json} json
+   * @memberof AdminController
+   */
+  static async changeUserStatus(request, response) {
+    const { id } = request.params;
+    const { status } = request.body;
+    try {
+      const user = await User.findOne({ where: { id } });
+      if (!user) {
```

Comment:
You can call the `findUser` helper function here, then you can save space by getting rid of `line 29 and 30`

Reply:
> You can call the `findUser` helper function here, then you can save space by getting rid of `line 29 and 30`

Done